app/helpers/container_summary_helper.rb

```diff
@@ -73,6 +73,13 @@ def textual_group_container_labels
     TextualGroup.new(_("Labels"), textual_key_value_group(@record.labels.to_a))
   end
 
+  def textual_group_annotations
+    if @record && @record.respond_to?(:custom_attributes)
+      annotations = @record.custom_attributes.select { |attribute| attribute.section == "annotations" }
```

Comment:
We have methods at the model level for separating these out, so I think this separation at the model level. See https://github.com/ManageIQ/manageiq/tree/master/app/models/container_group.rb#L20-L27

Reply:
And I think this is why custom_attributes is showing duplicate information.  If you look at the textual_miq_custom_attributes method it looks at everything (the superset).

Reply:
In that case, should I just remove annotations and labels from the nodes summary page and just leave the custom attributes dropdown? I believe that was the only summary page that had custom attributes instead of annotations and labels separately.

Reply:
I was actually thinking the other way around - maybe we just remove custom attributes?  Side q - maybe for @agrare, can we store custom attributes but for miq purposes?  If so, then I would think we'd keep custom attributes but filter out the labels/node_selectors/annoatations

Reply:
Yeah custom_attributes are used for a bunch of purposes, ":source => "EVM" is for MIQ internal purposes

Reply:
Sure, I can just remove it from UI so it only shows annotations and labels